Sources/MapboxMaps/Foundation/Extensions/Core/MapSnapshotOptions.swift

```diff
@@ -23,10 +29,14 @@ public struct MapSnapshotOptions {
     ///   - resourceOptions: Resource fetching options to be used by the
     ///         snapshotter. Default uses the access token provided by
     ///         `ResourceOptionsManager.default`
+    ///   - showsLogo:Flag that determines if the logo should be shown on the snapshot
+    ///   - showsAttribution:Flag that determines if attribution should be shown on the snapshot
```

Comment:
```suggestion
    ///   - showsLogo: Flag that determines if the logo should be shown on the snapshot
    ///   - showsAttribution: Flag that determines if attribution should be shown on the snapshot
```

Added spaces after colons